frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx

```diff
@@ -58,11 +72,20 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs }) => {
                         )}
                       </StackItem>
                       <StackItem className="co-truncate co-nowrap">
-                        <span className="co-resource-item">
+                        <span className="co-resource-item odc-gitops-details__env-section__co-resource-item">
                           <ResourceIcon kind="Project" />
                           <span className="co-resource-item__resource-name">{env.environment}</span>
                         </span>
                       </StackItem>
+                      <StackItem className="co-truncate co-nowrap">
+                        {env.environment && argocdLink && (
+                          <ExternalLink
+                            href={`${argocdLink.spec.href}/applications/${env.environment}-${appName}`}
+                            text="Argo CD"
+                            additionalClassName="odc-gitops-details__env-section__argocd-link"
+                          />
+                        )}
+                      </StackItem>
```

Comment:
@reginapizza I see that in the UXD spacing is changed so rather than adding `margins` you can add `CardBody` and `CardFooter` (https://www.patternfly.org/v4/components/card#basic) as the designs seems more like it.
 ```
<Card>
  <CardTitle className="odc-gitops-details__env-section__header">
    <Split style={{ alignItems: 'center' }} hasGutter>
      <SplitItem
        className="odc-gitops-details__env-section__title co-truncate co-nowrap"
        isFilled
      >
        {env.environment}
      </SplitItem>
      <SplitItem>
        <Label className="odc-gitops-details__env-section__timestamp" color="grey">
          <span style={{ color: 'var(--pf-global--palette--black-600)' }}>
            {env.timestamp}
          </span>
        </Label>
      </SplitItem>
    </Split>
  </CardTitle>
  <CardBody>
    <Stack>
      <StackItem className="co-truncate co-nowrap">
        {env.cluster ? (
          <ExternalLink
            additionalClassName="odc-gitops-details__env-section__url"
            href={env.cluster}
            text={env.cluster}
          />
        ) : (
          <div className="odc-gitops-details__env-section__url-empty-state">
            {t('devconsole~Cluster URL not available')}
          </div>
        )}
      </StackItem>
      <StackItem className="co-truncate co-nowrap">
        <span className="co-resource-item">
          <ResourceIcon kind="Project" />
          <span className="co-resource-item__resource-name">{env.environment}</span>
        </span>
      </StackItem>
    </Stack>
  </CardBody>
  {env.environment && argocdLink && (
    <CardFooter>
      <ExternalLink
        href={`${argocdLink.spec.href}/applications/${env.environment}-${appName}`}
        text="Argo CD"
        additionalClassName="odc-gitops-details__env-section__argocd-link"
      />
    </CardFooter>
  )}
</Card>            
```

Reply:
@reginapizza can you address this today itself? If there is any confusion on this let me know. Our feature freeze is this Friday and we need to get this in before that.

Reply:
@divyanshiGupta sorry for the delay today, there's been some issues with clusters. Adding `CardBody` makes the padding above the Argo CD link 24px instead of the 16px specified in the original design (you can see what that would look like below here). What would be the best way to decrease that spacing without changing the default styles of `CardBody`? 
![gitopsdetailspage](https://user-images.githubusercontent.com/50851526/100928390-22f26280-34b4-11eb-96bd-6a8df3f9aacc.png)

Reply:
Hmm then you can simply add the `argocd-link` in the body itself and add a spacing of 16px which I think will be `--pf-global--spacer--md` between `project` and `argocd-link` and remove the footer.